openapi.yaml

```
@@ -678,7 +678,6 @@ components:
       additionalProperties: false
```

I guess that we need to update that field as well?
```suggestion
      additionalProperties: true
```